documentation/docs/reference/services/Profile.md

```diff
@@ -199,20 +206,21 @@ GET /profile/search/?teamStatus=value&interests=value,value,value&limit=value
 
 Returns a list of profiles matching the filter conditions. 
 
-``teamStatus`` is a string matching the user's team status. Valid values for ``teamStatus`` are ``LOOKING_FOR_MEMBERS``, ``LOOKING_FOR_TEAM``, and ``NOT_LOOKING``.
+`teamStatus` is a string matching the user's team status. Valid values for ``teamStatus`` are ``LOOKING_FOR_MEMBERS``, ``LOOKING_FOR_TEAM``, and ``NOT_LOOKING``.
 
-interests is a comma-separated string representing the user's interests.
+`interests` is a comma-separated string representing the user's interests.
 
 - i.e if the user's interests are ["C++", "Machine Learning"], you can filter on this by sending ``interests="C++,Machine Learning"``
 
-If a ``limit`` parameter is provided, it will return the first matching ``limit`` profiles. Otherwise, it will return all of the matched profiles.
+If a `limit` parameter is provided, it will return the first matching `limit` profiles. Otherwise, it will return all of the matched profiles.
 
 Any users with the TeamStatus "NOT_LOOKING" will be removed.
```

Comment:
Just checked and verified. Not a thing. Instead of not allowing those profiles to be seen, maybe we can just add a disclaimer? If they wish to filter on the client they can through the `teamStatus` anyways. If you want to make an issue to change this behavior, feel free. For now, I'll just put a disclaimer.

See be59108.